odl/tomo/backends/astra_setup.py

```diff
@@ -339,7 +347,7 @@ def astra_projection_geometry(geometry):
             'parallel3d', det_width, det_height, det_row_count,
             det_col_count, angles)
 
-    elif isinstance(geometry, CircularConeFlatGeometry):
+    elif False and isinstance(geometry, CircularConeFlatGeometry):
```

Comment:
One important case is this in the astra doc for FDK (if we want to add an inverse):

[astra::FDK_CUDA](http://sourceforge.net/p/astra-toolbox/wiki/FDK_CUDA/)

> Supported geometries: only cone. (cone_vec is not supported.)